CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ accidentally triggering the load of a previous DB version.**
 * #3899 Fix segfault in Continuous Aggregates
 * #4225 Fix TRUNCATE error as non-owner on hypertable
 * #4259 Fix logic bug in extension update script
+* #4236 If there is a compressed hypertable with a segmentby column that uses a non-default collation, and there is an index on the compressed chunk on this column, and a query orders on this column, the order of results could be wrong. This is fixed for newly created hypertables, and the previously created tables continue to be affected by this problem. The root cause is that the segmentby columns of compressed chunks didn't have the proper collation. If your database is affected, you will receive a warning during upgrade. To fix, disable compression on the affected table and then enable it again.
```

Comment:
Would suggest something a little less verbose. For example: "Fix collation issue with compressed hypertable"

Reply:
This is almost what I had, but @sven wanted something more verbose. Maybe I have to put this verbose description into the original PR so that it's at least somewhere, and keep the terse version in the changelog.

Reply:
It depends a little on the purpose of the comment, but IMHO the lines in the list should be brief to make it easy to browse.

In general, I think these kinds of comments are most usable in the issue that the commit is fixing since it allow you to add comments even after the commits are pushed, allow you to correct mistakes and have several PRs that reference the issue, etc., Since we are referencing the PRs in our changelog messages, it should probably be in the referenced PR so that it is easy to find.

You might also consider adding it as a comment in the code.

Reply:
I added the verbose description to the original PR, and also some comments to my other compression PR.
Changed changelog to 'Fix potential wrong order of results for compressed hypertable with a non-default collation', does this sound OK?